pkg/apis/submariner.io/v1/types.go

```diff
@@ -344,3 +344,51 @@ type GlobalIngressIPList struct {
 
 	Items []GlobalIngressIP `json:"items"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:shortName="gwrt"
+
+type SubmarinerGwRoute struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	SubmarinerRoutePolicySpec SubmarinerRoutePolicySpec `json:"submarinerRoutePolicySpec"`
```

Comment:
Should this be `SubmarinerGwRouteSpec`?

Reply:
I thought we will reuse since fields are the same.